cli/cmd/testdata/inject_emojivoto_deployment_config_overrides.golden.yml

```diff
@@ -23,6 +22,7 @@ spec:
         linkerd.io/created-by: linkerd/cli dev-undefined
         linkerd.io/identity-mode: default
         linkerd.io/proxy-version: override
+        viz.linkerd.io/disable-tap: "true"
```

Comment:
I'd like to see this removed since this depends on an extension. Since we are testing the core install, this annotation should have no significance in testing.